### PR TITLE
Add glbinding 2.1.3-p0 package

### PR DESCRIPTION
**Pull request by NeroBurner**

### CODE REVIEW

File: docs/packages/pkg/glbinding.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+.. _pkg.glbinding:
+
+glbinding
+=========
+
+-  `Official <https://github.com/cginternals/glbinding>`__
+-  `Hunterized <https://github.com/hunter-packages/glbinding>`__
+-  `Example <https://github.com/ruslo/hunter/blob/master/examples/glbinding/CMakeLists.txt>`__
+-  Added by `NeroBurner <https://github.com/NeroBurner>`__ (`pr-N <https://github.com/ruslo/hunter/pull/N>`__)
```
Comment (ruslo) [inline docs/packages/pkg/glbinding.rst]: `s,pr-N,pr-1073,`

File: examples/glbinding/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+find_package(glbinding CONFIG REQUIRED)
+
+# Test double library creation
+find_package(glbinding CONFIG REQUIRED)
+
+add_executable(foo foo.cpp)
+target_link_libraries(foo glbinding::glbinding)
+
+# glbinding needs at least this c++11 feature
+target_compile_features(foo PRIVATE cxx_defaulted_move_initializers)
```
Comment (ruslo) [inline examples/glbinding/CMakeLists.txt]: C++11 will be added by toolchain. Not needed.

Comment (NeroBurner) [reply examples/glbinding/CMakeLists.txt]: I'd like to keep something in for future reference. When someone is using this package in his project he will need to enable this compile-feature or C++11. Not everyone is using a toolchain to enable C++11

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2510
* https://travis-ci.org/ingenue/hunter/builds/283648621?utm_source=github_status&utm_medium=notification

**Comment by NeroBurner** [comment]
tests succeeded :)

what shall I do because of the compile feature `cxx_defaulted_move_initializers`?

**Comment by ruslo** [comment]
> what shall I do because of the compile feature cxx_defaulted_move_initializers?

I've already finished testing so I guess nothing. My point is that we don't need it in examples because C++11 standard is set by toolchain while testing. Also setting C++ standard locally may cause ODR problems, it should be set globally:
* http://cgold.readthedocs.io/en/latest/tutorials/libraries/symbols/odr-global.html

Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.2511
* https://travis-ci.org/ingenue/hunter/builds/283683742?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.19.118

**Pull request merged**